resource-types/VSCodeRemoteDevelopment/awssamples-devtools-devinstance.json

```diff
@@ -0,0 +1,73 @@
+{
+    "typeName": "AWSSamples::Devtools::Devinstance",
+    "description": "An example resource schema demonstrating some basic constructs and validation rules.",
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git",
+    "properties": {
+        "UID": {
+            "description": "The ID of the developer instance",
+            "type": "string"
+        },
+        "InstanceType": {
+            "description": "The EC2 Instance Type of developer instance. You can only select instance types that support hibernation",
+            "type": "string",
+            "default": "m4.xlarge",
+            "enum": ["c4.large", "c4.xlarge", "c4.2xlarge", "c4.4xlarge", "c4.8xlarge", "c5.large", "c5.xlarge", "c5.2xlarge", "c5.4xlarge", 
```

Comment:
@PatMyron I'd remove the enum. Is it sensible to keep the default, or would you remove this as well, as it could be invalid at some point in the future?